Streamable/scripts/scr_string/scr_string.gml

```diff
@@ -0,0 +1,17 @@
+function string_pos_any()
```

Comment:
there's a `string_pos()` [built-in function](https://manual.yoyogames.com/#t=GameMaker_Language%2FGML_Reference%2FStrings%2Fstring_pos.htm) you could use here and then just loop the arguments:

```
var f = 0
for (var i = 1; i < argument_count; i++) {
	f = string_pos(argument[i], argument0)
	if f break
}
return f
```
(yeah, it's a little condensed)

Reply:
The way I do it finds the first character that matches any of them; this matches the first argument